docs/ilivalidator.rst

```diff
@@ -241,11 +241,12 @@ Aufruf-Syntax
 
 ``java -jar ilivalidator.jar [Options] [file]``
```

Comment:
hier schon andere Syntax benutzen (wie in man Pages), so dass man erkennt es sind mehrere files möglich